dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/user/AuthenticatedUser.java

```diff
@@ -104,6 +104,9 @@
     @Column(nullable = true)
     private Timestamp lastApiUseTime;   // last API use with user's token
 
+    @Column
+    private String notificationsLanguage;
```

Comment:
Now when we register new user we would end up with this field being null. I think we add a default value here it would be less error prone `notificationsLanguage = 'en' or `notificationsLanguage = Locale.ENGLISH`